### DISCUSSION TITLE
https://socket.io/ can not be accessed from Mainland China

### DISCUSSION THREAD

**Discussion by qiulang** [socketio/socket.io#3972] [Q&A] [answered]
My colleagues complained that they can not access https://socket.io/  from Mainland China. 
I remember we could access https://socket.io/  in 2.x version.  So what changes ?

**Comment by darrachequesne** [comment, marked as answer]
Hi! Thanks for the heads-up.

It seems Vercel (which hosts the website) has had some issues with China recently: https://www.vercel-status.com/incidents/r758bhbklgfd

It seems to be fixed though. Is there a way to test it?

**Reply by qiulang** [reply]
I am in China, I can test it. 
But unfortunately I still can not access  https://socket.io/ without vpn.

**Reply by darrachequesne** [reply]
I'm not familiar with this kind of problem, is there anything we can do on our side?

**Reply by qiulang** [reply]
Probably not much. It is just a shame that we can't access https://socket.io/.  Too bad but thanks for replying my inquiry

**Reply by qiulang** [reply]
Hi I am happy to report that https://socket.io/ is accessible from Mainland China! What have you guys done?

Anyway, thanks a lot!
